packages/@react-spectrum/upgrade-cli/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@react-spectrum/upgrade-cli",
+  "version": "0.0.1",
+  "private": true,
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "bin": {
+    "upgrade-react-spectrum": "./dist/index.js"
+  },
```

Comment:
```suggestion
"bin": "./dist/index.js",
```

Reply:
or we could give this an S1-to-S2-specific name so we can utilize this package for additional codemods in the future.

Reply:
I wasn't sure how it worked with scopes. What would the cli command be for `@react-spectrum/upgrade-cli`? Just `upgrade-cli`? Not sure that's specific enough.

Reply:
sure, you can change it in a separate pr if you like